esi_leap/tests/api/controllers/v1/test_utils.py

```diff
@@ -656,6 +476,49 @@ def test_check_resource_lease_admin_bad_time(self,
         self.assertIsNone(parent_lease_uuid)
 
 
+class TestOfferPolicyAndRetrieveUtils(testtools.TestCase):
+
```

Comment:
Incredibly nitpicky style detail but is there a reason why there's a newline after the class/function declarations in these test cases?

Reply:
I think it's because what a lot of OpenStack tests do, and because our linter doesn't complain :)